build/workers.props

```diff
@@ -0,0 +1,44 @@
+<Project>
+
+  <Import Project="$(MSBuildThisFileDirectory)python.props" />
+
+  <!-- Workers which are bundled into the product. -->
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Functions.DotNetIsolatedNativeHost" Version="1.0.9" />
+    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="2.14.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="3.10.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7.0" Version="4.0.3148" />
+    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7.2" Version="4.0.3220" />
+    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7.4" Version="4.0.3219" />
+  </ItemGroup>
+
+  <Target Name="RemovePowershellWorkerRuntimes" BeforeTargets="AssignTargetPaths" Condition="'$(RuntimeIdentifier)' != '' AND $(RuntimeIdentifier.StartsWith(win))">
```

Comment:
Might be overkill, but this makes me think about having a file for each worker, all imported by this one. We could keep all of the worker specific logic there and code ownership would also be more granular.

Reply:
Done